dependencies/Boost/gcc_5_no_cxx11.patch

```diff
@@ -0,0 +1,37 @@
+From eec808554936ae068b23df07ab54d4dc6302a695 Mon Sep 17 00:00:00 2001
```

Comment:
@lefticus I applied this patch to get Boost 1.55 to build with gcc 5.2 on RedHat 6.6.  Do you see any issues applying this patch on Ubutnu too?

@axelstudios @kbenne FYI